ui/app/components/transform-show-transformation.js

```diff
@@ -0,0 +1,8 @@
+import RoleEdit from './role-edit';
+
+export default RoleEdit.extend({
```

Comment:
You're totally right, we were using RoleEdit because it contains some methods that we need, but I'll pull it out into a separate component that all of transform can extend from.